Index: .idea/.idea.Solid_Principle/.idea/indexLayout.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.idea.Solid_Principle/.idea/indexLayout.xml b/.idea/.idea.Solid_Principle/.idea/indexLayout.xml
new file mode 100644
--- /dev/null
+++ b/.idea/.idea.Solid_Principle/.idea/indexLayout.xml
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="UserContentModel">
+    <attachedFolders />
+    <explicitIncludes />
+    <explicitExcludes />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/.idea.Solid_Principle/.idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.idea.Solid_Principle/.idea/.gitignore b/.idea/.idea.Solid_Principle/.idea/.gitignore
new file mode 100644
--- /dev/null
+++ b/.idea/.idea.Solid_Principle/.idea/.gitignore
@@ -0,0 +1,13 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Rider ignored files
+/.idea.Solid_Principle.iml
+/modules.xml
+/contentModel.xml
+/projectSettingsUpdater.xml
+# Editor-based HTTP Client requests
+/httpRequests/
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
Index: .idea/.idea.Solid_Principle/.idea/.name
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.idea.Solid_Principle/.idea/.name b/.idea/.idea.Solid_Principle/.idea/.name
new file mode 100644
--- /dev/null
+++ b/.idea/.idea.Solid_Principle/.idea/.name
@@ -0,0 +1,1 @@
+Solid_Principle
\ No newline at end of file
