Index: RocketLauncher.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/RocketLauncher.cs b/RocketLauncher.cs
new file mode 100644
--- /dev/null	(date 1747769337816)
+++ b/RocketLauncher.cs	(date 1747769337816)
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Solid_Principle
+{
+    internal class RocketLauncher
+    {
+        public void LaunchRocket()
+        {
+            Console.WriteLine("Rocket launched!");
+        }
+
+    }
+}
